index.js

```diff
@@ -124,19 +122,28 @@ exports.CommanderError = CommanderError;
 function Command(name) {
   this.commands = [];
   this.options = [];
-  this._execs = new Set();
   this._allowUnknownOption = false;
   this._args = [];
   this._name = name || '';
   this._optionValues = {};
   this._storeOptionsAsProperties = true; // backwards compatible by default
   this._passCommandToAction = true; // backwards compatible by default
   this._actionResults = [];
+  this._actionHandler = undefined;
```

Comment:
I'm sorry if I sound picky.
Here we 'define' it.So I think it's not 'undefined'.
I think 'null' is better, but how about?

Reply:
I did feel "defining" it to be "undefined" was a bit odd. I was really just adding it to be clear it was a property. I'll do a little reading on null vs undefined conventions. Good question.

Reply:
Changed all to null except the boolean that is lazy initialised where undefined felt better (and added a comment for that one).